src/pages/guides/using/troubleshooting.md

```diff
@@ -0,0 +1,18 @@
+# Troubleshooting
+
+If you're having trouble with your actions or activations, here are some common issues and solutions.
+
+## Exceeding local storage
+
+To improve performance and reduce the number of cold-start, activations for the same action are scheduled on the same K8s pod up to action concurrency limits.
+However, due to this approach activations will share resources available to them such as memory, CPU and local storage.
+The local storage size is not configurable and has a limit of [600Mb](./system_settings.md). As pods are getting recycled every [10 minutes](/src/pages/guides/index.md#zoom-in--what-is-happening-when-actions-are-invoked) heavily used actions can fill up the local storage and cause the activation to fail.
+
+The error message that indicates exceeding local storage is:
+```
+The pod exceeded its local storage and was evicted with X in flight activations
+```
+
+The most common reasons for exceeding local storage are:
+ - Writing too many log messages
+ - Log messages are heavy, for instance include entire JavaScript or JSON objects
```

Comment:
> Also 600MB and 10min are configureable and prone to change we should point to where they are listed so we can change them in one place.

I made references to the 600MB config, if it's not listed there it will take an extra effort to find the exact limits. Or it can be something like "local storage is limited".
10 minutes is mentioned in multiple places just as the description, but not as some configuration.

> I think we're getting into intricacies of the system that customers don't necessarily need to know and this can raise more questions.

From my experience answering this question multiple times, almost in all cases there was a follow-up on why the storage can be exceeded and how actions are scheduled. I think we should explain the technical details in the public doc as well, especially since it's no different from other similar technical docs that we already have. The target audience for public docs is developers since they write action code and should be aware of any limitations.

Reply:
I stand to my suggested change. It addresses the problem and offers solutions. 
btw. it's not only logging that leads to "The pod exceeded its local storage ..." it's application utilization of disk space too.

Reply:
I already changed it)